session1/3-challenge/1_arithmetic.rb

```diff
@@ -9,5 +9,6 @@
 # arithmetic1(-6)     # => -50
 
 def arithmetic1(n)
+	(n*5)-20
```

Comment:
George,
I don't think you need the parenthesis - multiplication operation takes precedence over subtraction.

Igor

Reply:
Absolutely right. This is not great form but an old habit from my hand written maths days. Only there for readability. Thanks for looking over the code.